Xamarin.Forms.Core/Profiler.cs

```diff
@@ -0,0 +1,123 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Diagnostics;
+using System.IO;
+using System.Reflection;
+using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices;
+using System.Threading;
+using System.Threading.Tasks;
+using Xamarin.Forms.Internals;
+
+namespace Xamarin.Forms.Internal
+{
+	[EditorBrowsable(EditorBrowsableState.Never)]
+	public struct Profile : IDisposable
+	{
+		const int Capacity = 1000;
+		const string PROFILE = "PROFILE";
+		public struct Datum
```

Comment:
Could you add the attribute `DebuggerDisplay` to Datum for more simple check in debug?
```C#
[DebuggerDisplay("{Name,nq} {Id} {Ticks}")]
```